### PR TITLE
Fix/default wakeup override bug 49 release

### PR DESCRIPTION
**Pull request by cjswedes**
This is an alternative solution suggfested by @greens to #935. It seems better because it achieves the same coverage of tcases where a battery powered device is using a subdriver, but it will be easier to revert.

### CODE REVIEW

File: drivers/SmartThings/zwave-sensor/src/aeotec-multisensor/multisensor-6/init.lua [hunk 1]
```diff
@@ -34,7 +34,8 @@ local function wakeup_notification(driver, device, cmd)
     device:set_field("__wakeup_interval_get_sent", true)
   end
   device:send(Configuration:Get({parameter_number = PREFERENCE_NUM}))
-  device:refresh()
+  local version = require "version"
+  if version.api ~= 6 then device:refresh() end
```
Comment (cjswedes) [inline drivers/SmartThings/zwave-sensor/src/aeotec-multisensor/multisensor-6/init.lua:38]: I ended up doing it this way along the way, but I could instead exclude the multisensor 6 and 7 in the `apiv6_bugfix` can_handle function like I did for a few other devices that have their own wakeup_notification handler. Looking at it this morning it seems better to do this to keep the bugfix local to its own subdriver, so I will go forward with that.

File: drivers/SmartThings/zwave-sensor/src/test/test_enerwave_motion_sensor.lua [hunk 2]
```diff
@@ -81,6 +81,12 @@ test.register_coroutine_test(
           mock_sensor,
           Battery:Get({})
       ))
+      --This extra battery get is because of apiv6 default handler override sub driver
+      --It was left in so when we remove that subdriver, a new bug isn't created
```
Comment (cjswedes) [inline drivers/SmartThings/zwave-sensor/src/test/test_enerwave_motion_sensor.lua:85]: We could specifically avoid these if we wanted to, but I thought that it opened us up to extra bugs. Like the multisensor case, I will try to exclude this device from the bugfix with fingerprints.

**Review by greens — APPROVED** [review]

**Comment by greens** [comment]
should probably squash these btw

**Comment by greens** [comment]
```
Checking drivers/SmartThings/zwave-sensor/src/apiv6_bugfix/init.lua 1 warning

    drivers/SmartThings/zwave-sensor/src/apiv6_bugfix/init.lua:11:7: (W211) unused variable SENSATIVE_STRIP_FP
```

**Pull request merged**